### DISCUSSION TITLE
Issue in path specfication while  executing forge script.

### DISCUSSION THREAD

**Discussion by MohiniVasisth** [Cyfrin/foundry-full-course-cu#1217] [Q&A]
Hi sir , Can you please tell me that how to fix this issue .
![Screenshot (88)](https://github.com/Cyfrin/foundry-full-course-f23/assets/98387326/9431dae9-f547-4114-80df-4e67a088ce53)

**Comment by xyz899** [comment]
Hi,

Simple when specifying a path, don't use backlashes like this : 
> `\`

 use : 
> `/`

Secondly, precise correctly your paths : there's a gap in Script.sol/ thirdly : CLI like Powershell or Bash are used for navigating between directories (folders), not for opening files like Script.sol. I would advise if you can't do this properly don't start by doing web3 development, start by learning computer science and how to use cli, linux and text editors. You need to know this some high or low level languages learn webapp and web2 before learning web3
